templates/confidential-computing/index.html

```diff
@@ -55,164 +68,154 @@ <h2>Benefits of Ubuntu for confidential computing</h2>
+            </p>
+          </div>
+          <div class="row u-sv3">
+            <div class="p-image__background--3-2 u-align--center u-vertically-center">
+              {{ image(url="https://assets.ubuntu.com/v1/480ebd6e-how-confidential-vms.png",
+                alt="",
+                width="1200",
+                height="800",
+                hi_def=True,
+                loading="auto|lazy"
+                ) | safe
+              }}
+            </div>
+          </div>
+          <div class="row">
+            <ol class="p-list--divided">
+              <li class="p-list__item p-heading-icon__title p-heading--5">Isolation</li>
+              <div class="col-6 col-medium-3 u-sv3">
+                <p>
+                  Confidential computing capable CPUs are equipped with an AES hardware memory encryption engine, which encrypts data when it is written to system memory, and decrypts it when read. The encryption key itself is stored in the hardware root of trust and is never exposed to the platform’s system software.
+                </p>
+              </div>
+              <li class="p-list__item p-heading-icon__title p-heading--5">Remote attestation</li>
+              <div class="col-6 col-medium-3">
+                <p>
+                  When a confidential VM is launched, its integrity is verified and its initial code and data are measured by a hardware root of trust. This ensures they have not been tampered with. The measurement is cryptographically signed and can be attested to a remote verifier
+                </p>
+              </div>
+            </ol>
+          </div>
+        </div>
+      </div>
+    </section>
 
     <section class="p-strip u-no-padding--top">
+      <hr class="p-rule is-fixed-width" />
       <div class="row">
-        <div class="col-6 col-medium-6">
-          <h2>Enterprise-grade confidential VMs on all major public clouds</h2>
+        <div class="col-12 u-sv3">
+          <h2>Launch enterprise-grade confidential VMs on all major public clouds</h2>
         </div>
-        <div class="col-6 col-medium-6">
+        <div class="col-9 col-start-large-4 col-start-medium-2">
+          <hr class="p-rule--muted" />
           <div class="row">
             <div class="col-2 col-medium-3 col-small-2 u-pad-logo">
               <p class="u-no-margin--bottom">
-                {{ image(url="https://assets.ubuntu.com/v1/dbab00f7-Microsoft_Azure.svg",
-                                alt="Azure logo",
-                                height="22",
-                                width="22",
-                                hi_def=True,
-                                loading="lazy") | safe
+                {{ image(url="https://assets.ubuntu.com/v1/aa515363-azure-logo-updated.svg",
+                  alt="Azure logo",
+                  width="568",
+                  height="56",
+                  hi_def=True,
+                  attrs={"style": "max-width: 100px;"},
```

Comment:
The image needs to be resized, but it should be done in the width and height attributes 
```suggestion
```